docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 5
+title: Tokens Overview
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar: 1) assets issued by Stellar accounts (Stellar assets can have a [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation), and 2) as a [custom token](./token-interface.mdx) created by a smart contract.
+
+Several factors (such as project-specific requirements and network capabilities) can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
+
+However:
+
+### TL;DR
+
+If possible, we recommend issuing a Stellar asset and (if needed) using the SAC to interact with that asset in smart contracts. More on why below.
+
+## Issuing assets on Stellar
+
+Stellar is first-class in asset tokenization — issuing an asset can be done in [just one single transaction](https://developers.stellar.org/docs).
+
+Stellar’s transactions are fast and cost-effective, making the network great for remittances and micropayments. It also has built-in features for compliance, asset management, and auditing. If you are just looking to perform simple transfers, issuing assets on Stellar has all the needed capabilities.
+
+Stellar assets:
+
+Are already widely supported in the Stellar ecosystem and other ecosystems (such as Stellar USDC on Coinbase). Benefit from [anchors](../../learn/encyclopedia/anchors.mdx), the bridges between the Stellar network and traditional financial systems. Explore the global [Stellar anchor directory](https://anchors.stellar.org/) for further details. Give the issuer granular control over asset management with features that allow the issuer to [publish asset information](../../issuing-assets/publishing-asset-info.mdx), [ensure compliance](../../issuing-assets/anatomy-of-an-asset.mdx#compliance), and [regulate asset supply](../../issuing-assets/control-asset-access.mdx#limiting-the-supply-of-an-asset).
+
+Assets issued on the Stellar network are accessible to smart contracts with the use of that asset’s Stellar Asset Contract (SAC).
+
+### Stellar Asset Contract
+
+The Stellar Asset Contract (SAC) is compiled into the protocol layer and allows smart contracts to interact with assets issued on Stellar. Every Stellar asset has reserved a SAC on the network that can be deployed by anyone who wants to interact with the asset from a contract. The SAC has access to all account (for XLM) and trustline (for all other assets) balances as well as smart contract token balances.
```

Comment:
Couple tweaks:
- Use builtin for built in contracts. It's the vocab we've been trying to use.
- The last sentence is a little confusing, I thought it was trying to say something else.

```suggestion
The Stellar Asset Contract (SAC) is built into the protocol layer and allows smart contracts to interact with assets issued on Stellar. Every Stellar asset, including the native asset XLM, has reserved a SAC on the network that can be deployed by anyone who wants to interact with the asset from a contract.
```

Reply:
I'd also change the "reserved" language as well. Maybe something like "A single instance of the SAC can be deployed for every Stellar asset by anyone who wants to interact with the asset from a contract"

Reply:
I really like the "two interfaces to the same thing" analogy between non-smart-contract operations and the SAC. Maybe we can add this point here as well.